frontends/election-manager/test/render_in_app_context.tsx

```diff
@@ -119,7 +119,8 @@ export function renderInAppContext(
     <AppContext.Provider
       value={{
         castVoteRecordFiles,
-        electionDefinition,
+        electionDefinition:
+          electionDefinition === 'NONE' ? undefined : electionDefinition,
```

Comment:
I needed a way to explicitly make the `electionDefinition` `undefined` in tests since passing `undefined` causes the default to get used:

https://github.com/votingworks/vxsuite/blob/cfae96a2ce960139e76d7500a904ed345c6999b6/frontends/election-manager/test/render_in_app_context.tsx#L85